resources/5/ajax.js

```diff
@@ -0,0 +1,223 @@
+    },
+    body: "code=" + code + "&op=create&name=" + nameStr + "&height=" + height + "&weight=" + weight
+  });
+
+  let data = await response.text();
+  if (data > 0) {
+    str = "Sikeresen létrehozva";
+  }
+  else {
+    str = "Létrehozás sikertelen!";
+    return;
+  }
+
+  document.getElementById("createResult").innerHTML = str;
+  document.getElementById("name1").value = "";
+  document.getElementById("height1").value = "";
+  document.getElementById("weight1").value = "";
+  read();
+}
+
+async function getDataForId() {
+  let id = document.getElementById("idUpd").value;
+  if (id == "") {  //filter needless API calls
+    return;
+  }
+
+  let response = await fetch(url, {
+    method: 'post',
+    cache: 'no-cache',
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded',
+    },
+    body: "code=" + code + "&op=read"
+  });
+
+  let data = await response.text();
+  let parsed_data = null;
+  try {
+    parsed_data = JSON.parse(data);
+  } catch (jsonError) {
+    console.error("Hiba: A válasz nem JSON formátumú!", data);
+    return;
+  }
+  for (let i = 0; i < parsed_data.list.length; i++)
+    if (parsed_data.list[i].id == id) {
+      document.getElementById("name2").value = parsed_data.list[i].name;
+      document.getElementById("height2").value = parsed_data.list[i].height;
+      document.getElementById("weight2").value = parsed_data.list[i].weight;
+    }
+}
+
+async function update() {
+  id = document.getElementById("idUpd").value;
+  if (id == "") {
+    document.getElementById("updateResult").innerHTML = "ID megadása kötelező!";
+    return;
+  }
+
+  let nameStr = document.getElementById("name2").value;
+  if (nameStr == "" || nameStr.length > 30) {
+    document.getElementById("updateResult").innerHTML = "Hibás név!";
+    return;
+  }
+
+  let height = document.getElementById("height2").value;
+  if (height == "" || height.length > 30) {
+    document.getElementById("updateResult").innerHTML = "Hibás magasság!";
+    return;
+  }
+
+  let weight = document.getElementById("weight2").value;
+  if (weight == "" || weight.length > 30) {
+    document.getElementById("updateResult").innerHTML = "Hibás súly!";
+    return;
+  }
+
+  let response = await fetch(url, {
+    method: 'post',
+    cache: 'no-cache',
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded',
+    },
+    body:
+      "code=" + code + "&op=update&id=" + id + "&name=" + nameStr + "&height=" + height + "&weight=" + weight
+  });
+
+  let data = await response.text();
+  if (data > 0) {
+    str = "Frissítés sikeres";
+  }
+  else {
+    str = "Frissítés sikertelen";
+    return;
+  }
+
+  document.getElementById("updateResult").innerHTML = str;
+  document.getElementById("idUpd").value = "";
+  document.getElementById("name2").value = "";
+  document.getElementById("height2").value = "";
+  document.getElementById("weight2").value = "";
+  read();
+}
+
+async function deleteF() {
```

Comment:
F is not needed at the end of the functions name, please update it in the html  as well!